File: Linguagem de Programação I/Extras/Beecrowd_Extra/1837.c
#include <stdio.h>

int main() {
    int a, b, r, q;
    scanf("%d %d", &a, &b);
    for (r = 0; r < abs(b); r++) {
        if ((a - r) % b == 0) {
            q = (a - r) / b;
            break;
        }
    }
    printf("%d %d\n", q, r);
    return 0;
}

File: Linguagem de Programação I/Extras/Beecrowd_Extra/1837.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();

        int q = a / b;
        int r = a % b;

        if (r < 0) {
            r += Math.abs(b);
            q = (a - r) / b;
        }

        System.out.println(q + " " + r);
    }
}